definitions/npm/lolex_v2.x.x/flow_v0.64.x-/lolex_v2.x.x.js

```diff
@@ -25,7 +26,8 @@ declare module 'lolex' {
     cancelAnimationFrame: typeof cancelAnimationFrame;
     hrtime: typeof process.hrtime;
     nextTick: typeof process.nextTick;
-    performace: {
+    now: number;
+    performance?: {
```

Comment:
not sure about this one - but performance is not there unless it exists in the `global` it installs from. So I _think_ adding the `?` is correct.